ci.adoc

```diff
@@ -23,15 +23,16 @@ when referencing nodes, e.g. in the `Jenkinsfile`.
 
 We generally prefer to use the Docker Pipeline plugin for system dependencies in the `Jenkinsfile` but it's also possible to use some basic, pre-configured, tool installers.
 
-* `jdk7` A recent JDK7 version
-* `jdk8` A recent JDK8 version
+* `jdk7` A recent OpenJDK 7 version
```

Comment:
Is this still recommended? Java 8 should be a solid minimum, no?

Reply:
Does that mean that we should remove jdk7?